ballerina/tests/jms_queue_tests.bal

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+
+final MessageProducer queue1Producer = check createProducer(AUTO_ACK_SESSION, {
+    'type: QUEUE,
+    name: "test-queue-1"
+});
+final MessageConsumer queue1Consumer = check createConsumer(AUTO_ACK_SESSION, destination = {
+    'type: QUEUE,
+    name: "test-queue-1"
+});
+
+@test:Config {
+    groups: ["queue"]
+}
+isolated function testQueueWithTextMessage() returns error? {
+    string content = "This is a sample message";
+    TextMessage message = {
+        content: content
+    };
+    check queue1Producer->send(message);
+    Message? response = check queue1Consumer->receive(5000);
+    test:assertTrue(response is TextMessage, "Invalid message type received");
+    if response is TextMessage {
+        test:assertEquals(response.content, content, "Invalid payload");
+    }
+}
+
+@test:Config {
+    groups: ["queue"]
+}
+isolated function testQueueWithMapMessage() returns error? {
+    map<anydata> content = {
+        "user": "John Doe",
```

Comment:
Double quotes are not required for the fields in maps

Reply:
Fixed with [5f114c64f2a26a5c1b9f6bca7b54cc8cb64deb24](https://github.com/ballerina-platform/module-ballerinax-java.jms/pull/82/commits/5f114c64f2a26a5c1b9f6bca7b54cc8cb64deb24)